packages/chart/src/components/LineChart/components/LineChart.Shape.tsx

```diff
@@ -115,7 +115,7 @@ const LineChartShape = (props: LineChartShapeProps) => {
   if (config.lineDatapointStyle === 'always show') {
     return (
       <VisxShape
-        display={config.visual.lineDatapointSymbol === 'standard' && dataIndex !== 5}
+        display={config.visual.lineDatapointSymbol === 'standard' && dataIndex < 7}
```

Comment:
Can we use a variable for the number seven that describes what seven is? Then we can export it to the other areas that use this number.